support-workers/src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -31,7 +36,7 @@ case class EmailFields(
       DataExtensionName = dataExtensionName,
       SfContactId = userId.left.toOption.map(_.id),
       IdentityUserId = userId.right.toOption.map(_.id)
-    ).jsonString
+    ).asJson.toString
```

Comment:
```suggestion
    ).asJson.noSpaces
```
toString behaves slightly odd if the Json isn't a string I think and even though this isn't it's probably better for consistency not to use it

Reply:
good point, I'll change.
this was actually moved from line 13 above.   I  intended to change to spaces2, as that's what it calls under the covers, and it's more type safe as you mention.
Do you have a particular preference between nospaces and spaces2?

Reply:
![image](https://user-images.githubusercontent.com/7304387/94167664-a96f4000-fe84-11ea-8202-4846cc7d4ce3.png)

Reply:
I've done spaces2 for now, happy to change before merging though